### PR TITLE
Bar_1

### PR DESCRIPTION
**Pull request by Jiaqi-beep**
- NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/bar_1.ncl
- NCL Plots: https://www.ncl.ucar.edu/Applications/Images/bar_1_1_lg.png, https://www.ncl.ucar.edu/Applications/Images/bar_1_2_lg.png, https://www.ncl.ucar.edu/Applications/Images/bar_1_3_lg.png
- Issue: #337 
Figure_1
Figure_2
Figure_3

### CODE REVIEW

**Comment by Jiaqi-beep** [comment]
Having some problems with the size

**Comment by anissa111** [comment]
> Having some problems with the size

Hello, what's going on?

**Comment by Jiaqi-beep** [comment]
> > Having some problems with the size
> 
> Hello, what's going on?

The sizing wasn't right so I split subplots into three separate plots

File: Plots/Bar/NCL_bar_1.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+============
+This script illustrates the following concepts:
+   - Drawing bars instead of curves in an XY plot
+   - Changing the aspect ratio of a bar plot
+   - Drawing bars up or down based on a Y reference value
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/bar_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/bar_1_1_lg.png, https://www.ncl.ucar.edu/Applications/Images/bar_1_2_lg.png
+                         and https://www.ncl.ucar.edu/Applications/Images/bar_1_3_lg.png
```
Comment (anissa111) [inline Plots/Bar/NCL_bar_1.py:12]:
```suggestion
    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/bar_1_1_lg.png, https://www.ncl.ucar.edu/Applications/Images/bar_1_2_lg.png and https://www.ncl.ucar.edu/Applications/Images/bar_1_3_lg.png
```

While it's ugly to have one really long line with a bunch of urls, the docs don't generate correctly if a line break is added:

image

File: Plots/Bar/NCL_bar_1.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data
+ds = xr.open_dataset(gdf.get("netcdf_files/soi.nc"))
+date = ds.date
+dsoik = ds.DSOI_KET
+#dsoid = ds.DSOI_DEC
```
Comment (anissa111) [inline Plots/Bar/NCL_bar_1.py:32]:
```suggestion
```

Remove unnecessary line

**Review by anissa111 — CHANGES_REQUESTED** [review]
This looks good! See minor comments below, then I'll approve :)

**Review by michaelavs — APPROVED** [review]
This looks good to me!

**Review by anissa111 — APPROVED** [review]
This looks good!

**Pull request merged**